test/metabase/query_processor/middleware/remove_inactive_field_refs_test.clj

```diff
@@ -0,0 +1,155 @@
+(ns metabase.query-processor.middleware.remove-inactive-field-refs-test
+  (:require
+   [clojure.test :refer :all]
+   [metabase.lib.metadata.jvm :as lib.metadata.jvm]
+   [metabase.query-processor :as qp]
+   [metabase.query-processor.store :as qp.store]
+   [metabase.test :as mt]
+   [metabase.util :as u]
+   [toucan2.core :as t2]))
+
+(deftest ^:synchronized deleted-columns-test
+  (mt/test-driver :h2
```

Comment:
Is there a reason to be explicit about `:h2`? Do new linters enforce this or is it something else?

Reply:
I don't think this should execute for all drivers. Let me see if I can use some Cam keyword to make it run once. Checking the invariant might get trickier (but more explicit, maybe).